plugins/openshift/commands/router-show-info.md

```
@@ -0,0 +1,166 @@
+---
+description: Display router process information
+argument-hint: "[router-pod-name]"
+---
+
+## Name
+openshift:router-show-info
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)